pkg/spiceerrors/assert_off.go

```diff
@@ -0,0 +1,8 @@
+//go:build !ci
+// +build !ci
+
+package spiceerrors
+
```

Comment:
Add a comment that this is always a no-op for prod builds

Reply:
Added